day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,18 @@
+count = 10
+
+while count > 0
+  if count >= 2
+    puts "#{count} speckled frogs sat on a log
+    eating some most delicious bugs.
+    One jumped in the pool where its nice and cool,
+    then there were #{count - 1} speckled frogs."
+
+  else
+    puts "1 speckled frog sat on a log
+    eating some most delicious bugs.
+    One jumped in the pool where its nice and cool,
+    then there were no more speckled frogs!"
+    break
+  end
+  count -= 1
+end
```

Comment:
Good start here- When your count is 2, there is also a special case because of the pesky phrasing in the last line (was 1 more, not were 1 one more). Also, try to explore how you may leverage gets.chomp to gain user input to set your count at different numbers.